src/C1T2X_OBU.py

```diff
@@ -21,16 +21,17 @@
 
 from Networking.networking import UDP_NET
 
-# LOGGING
+### Logging
 c1t2x_logger = None
-# Get Logging Level
 LOGGING_LEVEL = logging.INFO
 logs_directory = os.path.join(os.getcwd(), "Logs")
+
 # IF: Check if the Logs directory does not exist
 if not os.path.exists(logs_directory):
 	# Create Logs directory
 	os.mkdir(logs_directory, 0o777)
```

Comment:
These permissions are probably too loose. Consider changing to `775`, which is the same permissions level for `/var/log`.